tests/test_convert_to_onnx.py

```diff
@@ -29,6 +30,12 @@
 TESTS = list(itertools.product(TORCH_DEVICE_OPTIONS, [True, False], [True, False]))
 TESTS_ORT = list(itertools.product(TORCH_DEVICE_OPTIONS, [True]))
 
+ON_AARCH64 = platform.machine() == "aarch64"
+if ON_AARCH64:
+    rtol, atol = 1e-1, 1e-2
```

Comment:
Any idea why ARM is less accurate with these tests? Floating point errors? I think it's fine to have this since the tests aren't about exacting accuracy with results.

Reply:
The discrepancy between the results generated on the ARM and x86 architectures primarily stems from the differing precision handling inherent to each architecture. Additionally, I have discovered that the matmul operation produces varying results when performed on these architectures.

Here is a demonstrative example:
```
a = torch.tensor([[[ 1.0000,  0.0000,  0.0000, -0.5000],
         [ 0.0000,  1.0000,  0.0000, -0.5000],
         [ 0.0000,  0.0000,  0.6667, -0.6667],
         [ 0.0000,  0.0000,  0.0000,  1.0000]]], device='cuda:0')
b = torch.tensor([[[ 1.,  0.,  0.,  0.],
         [-0., -1., -0.,  1.],
         [ 0.,  0.,  1.,  0.],
         [ 0.,  0.,  0.,  1.]]], device='cuda:0')
c = torch.tensor([[[1.0000, 0.0000, 0.0000, 0.5000],
         [0.0000, 1.0000, 0.0000, 0.5000],
         [0.0000, 0.0000, 1.5000, 1.0000],
         [0.0000, 0.0000, 0.0000, 1.0000]]], device='cuda:0')

print(a @ b @ c)
In x86:
tensor([[[ 1.0000,  0.0000,  0.0000,  0.0000],
         [ 0.0000, -1.0000,  0.0000,  0.0000],
         [ 0.0000,  0.0000,  1.0001,  0.0000],
         [ 0.0000,  0.0000,  0.0000,  1.0000]]], device='cuda:0')
In arm:
tensor([[[ 1.0000,  0.0000,  0.0000,  0.0000],
         [ 0.0000, -1.0000,  0.0000,  0.0000],
         [ 0.0000,  0.0000,  0.9998,  0.0000],
         [ 0.0000,  0.0000,  0.0000,  1.0000]]], device='cuda:0')
```

Reply:
The difference is a little surprising, it could be CUDA behaviour on ARM, differences in float32 accuracy, something in Pytorch, or a combination. I don't know if we can do anything about these so let's leave the tests, but if anyone asks why their ARM results are different we'll have something to discuss. Thanks!